esrally/mechanic/mechanic.py

```diff
@@ -176,8 +176,8 @@ def cluster_distribution_version(cfg, client_factory=client.EsClientFactory):
     :param client_factory: Factory class that creates the Elasticsearch client.
     :return: The distribution version.
     """
-    hosts = cfg.opts("client", "hosts")
-    client_options = cfg.opts("client", "options")
+    hosts = cfg.opts("client", "hosts")()
+    client_options = cfg.opts("client", "options")()
```

Comment:
I think it would be better to use an explicit property name here instead of `()`?